src/Microsoft.DocAsCode.Build.Common/MarkdownReader.cs

```diff
@@ -33,50 +37,118 @@ public static List<OverwriteDocumentModel> ReadMarkdownAsOverwrite(string baseDi
             };
         }
 
-        private static IEnumerable<OverwriteDocumentModel> ReadMarkDownCore(string file)
+        private static OverwriteDocumentModel ReadMarkDownCore(string filePath, YamlHtmlPart part)
```

Comment:
Since you refactored the logic, the method name doesn't reflect what it does. Consider a better name.

---
In reply to: [65309249](https://github.com/dotnet/docfx/pull/343#discussion_r65309249) [](ancestors = 65309249)